crates/bevy_ui/src/layout/ui_surface.rs

```diff
@@ -194,10 +194,11 @@ impl UiSurface {
     }
 
     /// Compute the layout for each window entity's corresponding root node in the layout.
+    #[cfg(all(feature = "bevy_render", feature = "bevy_text"))]
     pub fn compute_camera_layout<'a>(
         &mut self,
         camera: Entity,
-        render_target_resolution: UVec2,
+        render_target_resolution: bevy_math::UVec2,
         buffer_query: &'a mut bevy_ecs::prelude::Query<&mut bevy_text::ComputedTextBlock>,
         font_system: &'a mut CosmicFontSystem,
```

Comment:
```suggestion
        #[cfg(feature = "bevy_text")] buffer_query: &'a mut bevy_ecs::prelude::Query<
            &mut bevy_text::ComputedTextBlock,
        >,
        #[cfg(feature = "bevy_text")] font_system: &'a mut CosmicFontSystem,
```
\
Also need to add three feature gates to the body of the function too:

```rust
        for root_nodes in camera_root_nodes {
            self.taffy
                .compute_layout_with_measure(
                    root_nodes.implicit_viewport_node,
                    available_space,
                    |known_dimensions: taffy::Size<Option<f32>>,
                     available_space: taffy::Size<taffy::AvailableSpace>,
                     _node_id: taffy::NodeId,
                     context: Option<&mut NodeMeasure>,
                     style: &taffy::Style|
                     -> taffy::Size<f32> {
                        context
                            .map(|ctx| {
                                #[cfg(feature = "bevy_text")]
                                let buffer = get_text_buffer(
                                    crate::widget::TextMeasure::needs_buffer(
                                        known_dimensions.height,
                                        available_space.width,
                                    ),
                                    ctx,
                                    buffer_query,
                                );
                                let size = ctx.measure(
                                    MeasureArgs {
                                        width: known_dimensions.width,
                                        height: known_dimensions.height,
                                        available_width: available_space.width,
                                        available_height: available_space.height,
                                        #[cfg(feature = "bevy_text")]
                                        font_system,
                                        #[cfg(feature = "bevy_text")]
                                        buffer,
                                        _phantom: default(),
                                    },
                                    style,
                                );
                                taffy::Size {
                                    width: size.x,
                                    height: size.y,
                                }
                            })
                            .unwrap_or(taffy::Size::ZERO)
                    },
                )
                .unwrap();
```

Then layout should work again without `bevy_text`.